FEXCore/Source/Interface/IR/IR.json

```diff
@@ -1430,15 +1430,15 @@
         "DestSize": "RegisterSize",
         "NumElements": "RegisterSize / (ElementSize << 1)"
       },
-      "FPR = VUShl u8:#RegisterSize, u8:#ElementSize, FPR:$Vector, FPR:$ShiftVector": {
+      "FPR = VUShl u8:#RegisterSize, u8:#ElementSize, FPR:$Vector, FPR:$ShiftVector, i8:$RangeCheck": {
```

Comment:
aren't bools supposed to be indicated with i1, or are these synonymous?

Reply:
You're right, I should switch it over to i1.

Reply:
Switched to i1.